src/sentry/incidents/endpoints/organization_alert_rule_details.py

```diff
@@ -130,12 +144,119 @@ def remove_alert_rule(request: Request, organization, alert_rule):
         return Response("This rule has already been deleted", status=status.HTTP_400_BAD_REQUEST)
 
 
+@extend_schema_serializer(exclude_fields=["excludedProjects", "thresholdPeriod"])
+class OrganizationAlertRuleDetailsPutSerializer(serializers.Serializer):
+    name = serializers.CharField(max_length=64, help_text="The name for the rule.")
+    aggregate = serializers.CharField(
+        help_text="A string representing the aggregate function used in this alert rule. Valid aggregate functions are `count`, `count_unique`, `percentage`, `avg`, `apdex`, `failure_rate`, `p50`, `p75`, `p95`, `p99`, `p100`, and `percentile`. See [Metric Alert Rule Types under Create a Metric Alert Rule](/api/alerts/create-a-metric-alert-rule-for-an-organization/#metric-alert-rule-types) for valid configurations."
```

Comment:
I think the link is a little long, so we can split it between the title and page itself
```suggestion
        help_text="A string representing the aggregate function used in this alert rule. Valid aggregate functions are `count`, `count_unique`, `percentage`, `avg`, `apdex`, `failure_rate`, `p50`, `p75`, `p95`, `p99`, `p100`, and `percentile`. See **Metric Alert Rule Types** under [Create a Metric Alert Rule](/api/alerts/create-a-metric-alert-rule-for-an-organization/#metric-alert-rule-types) for valid configurations."
```